### DISCUSSION TITLE
Can CRC install a openshift 4.1x nightly build ?

### DISCUSSION THREAD

**Discussion by claudio4j** [crc-org/crc#3241] [General]
Can crc installs [openshift 4.11 nightly build](https://mirror.openshift.com/pub/openshift-v4/clients/ocp-dev-preview/4.11.0-0.nightly-2022-06-25-081133/) ?

**Comment by cfergeau** [comment]
> Can crc installs [openshift 4.11 nightly build](https://mirror.openshift.com/pub/openshift-v4/clients/ocp-dev-preview/4.11.0-0.nightly-2022-06-25-081133/) ?

If you want a nightly build, you need to use github.com/code-ready/snc and set `MIRROR=https://mirror.openshift.com/pub/openshift-v4/clients/ocp-dev-preview` and `OPENSHIFT_RELEASE_VERSION=4.11.0-0.nightly-2022-06-25-081133`

This is quite a tricky process though :-/
